code/game/machinery/doors/firedoor.dm

```diff
@@ -74,6 +74,10 @@
 
 	check_atmos()
 
+	if(prob(0.1) && icon == 'icons/obj/doors/Doorfireglass.dmi')
+		base_icon_state = "sus"
+		desc += " This one looks a bit sus..."
```

Comment:
speaks for itself almost honestly, think you need this?

Reply:
It's mostly there if by some miracle people examine open firelocks. The discovery of finding something cursed before it itself can reveal itself is pretty fun too.